bot/seasons/evergreen/magic_8ball.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import random
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+answers = [
+    "It is certain",
+    "It is decidedly so",
+    "Without a doubt",
+    "Yes definitely",
+    "You may rely on it",
+    "As I see it, yes",
+    "Most likely",
+    "Outlook good",
+    "Yes",
+    "Signs point to yes",
+    "Reply hazy try again",
+    "Ask again later",
+    "Better not tell you now",
+    "Cannot predict now",
+    "Concentrate and ask again",
+    "Don't count on it",
+    "My reply is no",
+    "My sources say no",
+    "Outlook not so good",
+    "Very doubtful",
+]
```

Comment:
This command was listed as part of the evergreen season, I've added this to the Halloween resources directory. Should I add this file to a new directory named `evergreen` in `resources` directory? Keeping the data a bit more seasonally organized. #12 seems to be a Halloween version of this command, this may cause confusion in the future.

Reply:
The resources for this cog should go into an evergreen resources folder, this is a separate cog from the functionality proposed in #12

Reply:
I apologize for that, it should be in evergreen, as @sco1 mentioned.